.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   build:
     docker: &image
-      - image: circleci/openjdk:11-jdk
+      - image: circleci/openjdk:8-jdk
```

Comment:
❓ Why is this being set back to java 8?

Reply:
I had to downgrade the java version in the repo to work in the Ion Source. I'm not sure if there is a better solution. I assumed I also needed to downgrade the version in CircleCI as well.

Reply:
I don't think this is something we want to do.

Reply:
❓ Do we need two versions of the api (java8 & java11)?

Reply:
I just tried that out, and it works with a few changes to the maven-compiler-plugin in the different modules, but it does add some complexity because the source and the service would need unique artifacts. `ion-source` would depend on `query-api-rest-jaxrscxf-client-stubs` and `query-api-rest-models-java8`, and `ion-search` would depend on `query-api-rest-spring-server-stubs` and `query-api-rest-models-java11`.

Reply:
If you decide to do this, you should probably use classifiers instead of including java8 and java11 in the artifact names.

Reply:
I split out this change into a new PR: https://github.com/connexta/ion-query-api/pull/40. Let's have any further discussion on that PR instead.

Reply:
Using different profiles for Java 8 and 11 builds, and a classifier like @figliold suggested above for the artifacts would probably be your safest bets, but I'm not sure how well our CI/CD pipelines would be able to support this. Probably something to check with @j-fontaine.
For now, could we just build the API using Java 8 and the service itself with Java 11?

Reply:
Moving convo to https://github.com/connexta/ion-query-api/pull/40/files#r352815158